code/style.css

```diff
@@ -1,7 +1,192 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
-  /* some styling here */
+ * {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
 }
+
+body {
+  font-family: 'Josefin Sans', sans-serif;
+  background-image: url("/img/skiingpicture.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: bottom;
+}
```

Comment:
In the future you could use the background shorthand
body {
  background: url("img_tree.png") no-repeat cover bottom;
}